dotnet-directory

```diff
@@ -49,14 +49,23 @@ print_sdk() {
 }
 
 print_framework() {
-    if [[ $# -eq 0 ]]; then
+    if [ "$1" != "dotnet" ] | [ "$1" != "aspnet" ]; then 
```

Comment:
Can you make this function take an argument that gets set to the shared directory name by the caller, like:

```
    --framework)
        print_framework "Microsoft.NETCore.App" "$@"
    ;;
    --aspnet)
        print_framework "Microsoft.AspNetCore.App" "$@"
    ;;
```